### ISSUE TITLE
Incorrect documentation for SQS.Queue.receive_messages & SQS.Client.receive_message

### ISSUE THREAD

**Issue by sparrowt** [boto/botocore#1610]
The documentation for both [SQS.Client.receive_message](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Client.receive_message) and [SQS.Queue.receive_messages](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Queue.receive_messages) has an incorrect list of `AttributeNames` under **Request Syntax**
```
    AttributeNames=[
        'All'|'Policy'|'VisibilityTimeout'|'MaximumMessageSize'|'MessageRetentionPeriod'|'ApproximateNumberOfMessages'|'ApproximateNumberOfMessagesNotVisible'|'CreatedTimestamp'|'LastModifiedTimestamp'|'QueueArn'|'ApproximateNumberOfMessagesDelayed'|'DelaySeconds'|'ReceiveMessageWaitTimeSeconds'|'RedrivePolicy'|'FifoQueue'|'ContentBasedDeduplication'|'KmsMasterKeyId'|'KmsDataKeyReusePeriodSeconds',
    ],
```

The issue is that it lists attributes of a _queue_, rather than of a _message_.

In both cases, the corresponding description under **Parameters** > **AttributeNames** has the correct list with further details of each, and is described as _"A list of s that need to be returned along with each message."_ (typo: `/s/attributes`), so it is just the **Request Syntax** that has the wrong list of attributes.

As far as I can tell this documentation is generated from [data/sqs/2012-11-05/service-2.json](https://github.com/boto/botocore/blob/a20591a4/botocore/data/sqs/2012-11-05/service-2.json) in `botocore`, hence filing the issue here rather than on the `boto3` repo:
 - [`ReceiveMessage`](https://github.com/boto/botocore/blob/a20591a4/botocore/data/sqs/2012-11-05/service-2.json#L204) has input shape based on [`ReceiveMessageRequest`](https://github.com/boto/botocore/blob/a20591a4/botocore/data/sqs/2012-11-05/service-2.json#L1076)...
 - ...whose [`AttributeNames`](https://github.com/boto/botocore/blob/a20591a4/botocore/data/sqs/2012-11-05/service-2.json#L1084) have shape [`AttributeNameList`](https://github.com/boto/botocore/blob/a20591a4/botocore/data/sqs/2012-11-05/service-2.json#L343) which is based on [`QueueAttributeName`](https://github.com/boto/botocore/blob/a20591a4/botocore/data/sqs/2012-11-05/service-2.json#L1002)

It seems that (instead of using `AttributeNames` > `AttributeNameList` > `QueueAttributeName`) the input shape for `ReceiveMessageRequest` needs to use something that ends up based on the list of attributes in [`MessageSystemAttributeName`](https://github.com/boto/botocore/blob/a20591a4/botocore/data/sqs/2012-11-05/service-2.json#L942) instead, which seems to be the correct list of _message_ attributes.

As per CONTRIBUTING.rst I am filing this issue without attempting to fix it, given the `service-2.json` files are generated "upstream based on our internal knowledge of the AWS services".

**Comment by joguSD** [comment]
It seems like you're correct here. The possible values listed in the operation documentation are correct, but the auto-generated values in the sample syntax are incorrect. As you've noted we pull in these values from the service model that's from the upstream service team. Unfortunately, this means that we won't be able to fix this directly in botocore. I've forwarded this issue along to the appropriate team.

Thanks for the report. Closing out the issue here as there's no actionable item for botocore.
